authorizerd.go

```diff
@@ -354,21 +354,30 @@ func (a *authority) AuthorizeAccessToken(ctx context.Context, tok, act, res stri
 }
 
 func (a *authority) authorize(ctx context.Context, m mode, tok, act, res string, cert *x509.Certificate) (Principal, error) {
-	var key string
+	var key strings.Builder
+	key.WriteString(tok)
+
+	if cert != nil {
+		key.WriteRune(':')
```

Comment:
Better to make the `':'` a constant/variable?

Reply:
Fixed
https://github.com/yahoojapan/athenz-authorizer/pull/75/commits/2f4da3ffc4091a1395b41a8d5c53b87131c92597